skynet/agent.py

```diff
@@ -1,15 +1,23 @@
 import asyncio
```

Comment:
Can we pl rename this file to haproxy_agent.py? The "agent" term is a bit overloaded in the AI space :-)